imu_topic: /livox/imu
lidar_topic: /livox/lidar
body_frame: body
world_frame: lidar
print_time_cost: false

lidar_filter_num: 3
lidar_min_range: 0.5
lidar_max_range: 30.0
scan_resolution: 0.15
map_resolution: 0.3

cube_len: 300
det_range: 60
move_thresh: 1.5

na: 0.01
ng: 0.01
nba: 0.0001
nbg: 0.0001

imu_init_num: 20
near_search_num: 5
ieskf_max_iter: 5

gravity_align: true
esti_il: false

ext_il: [-0.011, -0.02329, 0.04412, 0.0, 0.0, 0.0, 1.0] # x y z qx qy qz qw
ext_lc:
  [
    0.055301417087128005,
    -0.023940681099107203,
    -0.036510243108364156,
    0.5040772962309031,
    -0.5018066026651014,
    0.49854633904149553,
    -0.4955277598425481,
  ] # x y z qx qy qz qw

cam_width: 1280
cam_height: 720
cam_fx: 641.976318359375
cam_fy: 641.0658569335938
cam_cx: 641.5721435546875
cam_cy: 368.21832275390625
cam_d: [-0.05588280409574509, 0.06619580090045929, 0.0002959131670650095, 0.0008284428040497005, -0.021595485508441925]

lidar_cov_inv: 1000.0
